src/widgets/tooltip/Tooltip.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component, PropTypes } from 'react';
+import { Gateway } from 'react-gateway';
+import { getElementPosition } from './tooltipHelpers';
+import SimpleTooltip from './SimpleTooltip';
+
+const DELAY = 500;
```

Comment:
@adcpm Yes thanks. Delay value works accurately. The problem was with profile tooltip that was taking longer because we couldn't see loading time. Fixed!